src/main/resources/static/js/ei-rules.js

```diff
@@ -0,0 +1,10 @@
+jQuery(document).ready(function () {
+    var callback = {
+        success: function (responseData, textStatus) {
```

Comment:
error callback missing and log the error to the UI?

Reply:
It's now default behavior that if no callback error is defined it will try to parse the response as an error message.